### PR TITLE
Also allow relative url

### PR DESCRIPTION
**Pull request by skjnldsv**
Signed-off-by: John Molakvoæ (skjnldsv) <skjnldsv@protonmail.com>

### CODE REVIEW

File: src/components/ActionLink/ActionLink.vue [hunk 1]
```diff
@@ -75,11 +75,11 @@ export default {
 			default: '#',
 			required: true,
 			validator: value => {
-				// href is either an anchor or a valid url
+				// href is either an anchor or a valid url starting with a scheme or an absolute path
 				try {
 					return new URL(value)
 				} catch (error) {
-					return value.startsWith('#')
+					return value.startsWith('#') || value.startsWith('/')
```
Comment (ChristophWurst) [inline src/components/ActionLink/ActionLink.vue]: relative? :P

Comment (skjnldsv) [reply src/components/ActionLink/ActionLink.vue]: how could we check that? :thinking:

Comment (ChristophWurst) [reply src/components/ActionLink/ActionLink.vue]: no. a URL that start with `/` is *relative*. One that starts with a scheme or the host is *absolute*

Comment (ChristophWurst) [reply src/components/ActionLink/ActionLink.vue]: The PR description (and the source code comment) should say *Also allow relative url*

Comment (skjnldsv) [reply src/components/ActionLink/ActionLink.vue]: :see_no_evil:

**Pull request merged**